src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -42,6 +42,16 @@ typedef struct uct_ib_mlx5_srq_info {
     unsigned           tail;
 } uct_ib_mlx5_srq_info_t;
 
+#if defined(HAVE_DECL_IBV_MLX5_EXP_GET_QP_INFO)  || \
+    defined(HAVE_DECL_IBV_MLX5_EXP_GET_SRQ_INFO) || \
+    defined(HAVE_DECL_IBV_MLX5_EXP_GET_CQ_INFO)
+static void uct_ib_mlx5_obj_error(const char *obj_name)
+{
+    ucs_error("Failed to get mlx5 %s information. Please make sure the installed "
+              "libmlx5 version matches the one UCX was compiled with. UCX was "
```

Comment:
need to make message as short as possible:
"Failed to get mlx5 %s information. Please make sure the installed libmlx5 version matches the one UCX was compiled with (%s)"

Reply:
I'd get rid of "please" in error message as well